src/balena-utils.ts

```diff
@@ -146,6 +146,29 @@ export async function getReleaseByTags(
 	};
 }
 
+// https://www.balena.io/docs/reference/sdk/node-sdk/#balena.models.release.get
+export async function getReleaseVersion(releaseId: number): Promise<string> {
+	core.debug(`Getting raw version for release ID ${releaseId}`);
+
+	await balena.auth.loginWithToken(
+		core.getInput('balena_token', { required: true }),
+	);
+
+	return await balena.models.release
+		.get(releaseId, { $select: 'raw_version' })
+		.then((release) => {
```

Comment:
I don't like that we are mixing await with then because it's 2 different styles of handling promises so we should just use 1 (await btw).